radsat-sk/src/tasks/RSatelliteWatchdogTask.c

```diff
@@ -32,7 +32,7 @@ void SatelliteWatchdogTask(void* parameters) {
 
 		// TODO: implement petting satellite watchdogs
 
-		debugPrint("SatelliteWatchdogTask(): About to pet external satellite watchdogs.\n");
+		//debugPrint("SatelliteWatchdogTask(): About to pet external satellite watchdogs.\n");
 

```

Comment:
Throw pdbPetWatchdog in there for me, reduce the rate to 100ms and remove the debug print.